### PR TITLE
Add vault structure

### PR DESCRIPTION
**Pull request by argenos**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+To contribute, please download the latest version of this vault and open it in Obsidian. This is important to avoid duplicating content, and allows you to use the autocomplete when linking to other notes, searching and following our content guidelines.
+
+Because PKM is usually (as it name states) *highly* personal, we have added a few rules about how we use Obsidian features to preserve our sanity:
+
+- **Tags**: We mostly use tags to indicate the status of a note. You can find more details in the [[Tag glossary]]. 
+- **Files & Links**: Make sure that any time you add or edit a note, your settings are configured correctly. To sum up: 
+	- Configure *Location of new notes* to go the `08 - Seedbox` folder. 
+	- Links should be set to `Shortest path` and wikilinks must be enabled
+	- The *Default location for new attachments* should be the folder `09.01 - Note attachments`
```
Comment (kometenstaub) [inline CONTRIBUTING.md:25]: This needs to be 10.01

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+
+	![[file-and-link-settings.png]]
```
Comment (kometenstaub) [inline CONTRIBUTING.md:27]: The screenshot needs to be updated because of the attachments folder

**Review by kometenstaub — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+
+1. Create a new note (if you followed the steps in [[#Setup]] it should be created under [[08 - Seedbox]])
+2. Apply the [[T - New Note]] template and write! 
+3. Submit to GitHub
+
+### Adding or editing notes
+
+#### [[02 - Community plugins]] 
+
+- [[01.01 - Plugins|Community plugins]] - This folder is pre-populated with information from the [obsidian-releases]() repository. Here you can add more information about the plugin itself. For [[09.06 - How To]] do something with a plugin keep reading.
```
Comment (kometenstaub) [inline CONTRIBUTING.md:47]: [[09.02 - How To|how to]]

File: 09 - Digital Garden/09.01 - Topics/Begginer guides.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+---
+alias: 
+- 
+tags:
+- seedling
+---
+
+# Begginer guides
```
Comment (kometenstaub) [inline 09 - Digital Garden/09.01 - Topics/Begginer guides.md:8]: Typo; same typo in file name.

File: 02 - Community plugins/02 - Community plugins.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+## MOC
+
+%% Zoottelkeeper: Beginning of the autogenerated index file list  %%
+- [[02 - Community plugins/02.01 - Plugins/02.01 - Plugins|02.01 - Plugins]]
+- [[02 - Community plugins/02.02 - Categories/02.02 - Categories|02.02 - Categories]]
+%% Zoottelkeeper: End of the autogenerated index file list  %%
+
+## Contributing
+
+- [[01.01 - Plugins|Community plugins]] - This folder is pre-populated with information from the [obsidian-releases]() repository. Notes in this folder might have missing information (marked with the #placeholder tag)
```
Comment (kometenstaub) [inline 02 - Community plugins/02 - Community plugins.md:21]: [[02.01 - Plugins|Community plugins]]

File: 02 - Community plugins/02 - Community plugins.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+
+%% Zoottelkeeper: Beginning of the autogenerated index file list  %%
+- [[02 - Community plugins/02.01 - Plugins/02.01 - Plugins|02.01 - Plugins]]
+- [[02 - Community plugins/02.02 - Categories/02.02 - Categories|02.02 - Categories]]
+%% Zoottelkeeper: End of the autogenerated index file list  %%
+
+## Contributing
+
+- [[01.01 - Plugins|Community plugins]] - This folder is pre-populated with information from the [obsidian-releases]() repository. Notes in this folder might have missing information (marked with the #placeholder tag) 
+	- Click on [[09.06 - How To]] if you want to add how to do something with a plugin.
```
Comment (kometenstaub) [inline 02 - Community plugins/02 - Community plugins.md:22]: [[09.02 - How To]]

**Review by kometenstaub — CHANGES_REQUESTED** [review]

**Comment by argenos** [comment]
@kometenstaub thanks for catching all those broken links! 

@ericaxu we could merge this! I'll add the GitHub guide on a separate commit/PR (it's a much smaller change).

File: 00 - Meta/00 - Meta.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
+---
+alias:
```
Comment (kometenstaub) [inline 00 - Meta/00 - Meta.md:2]: I know that `alias` also works, but in the docs we only say to use `aliases`, so we shouldn't confuse contributors.
https://help.obsidian.md/How+to/Add+aliases+to+note#Set+aliases

**Review by kometenstaub — CHANGES_REQUESTED** [review]
I also checked it out locally and took a quick look, apart from changing `alias` to `aliases` I didn't find anything else.
(`alias` is used in 86 notes, `aliases` in 441)

File: 02 - Community plugins/02.01 - Plugins/02.01 - Plugins.md [hunk 8]
```diff
@@ -132,7 +132,7 @@ tags:
 - [[02 - Community plugins/02.01 - Plugins/obsidian-code-copy|obsidian-code-copy]]
 - [[02 - Community plugins/02.01 - Plugins/obsidian-codemirror-options|obsidian-codemirror-options]]
 - [[02 - Community plugins/02.01 - Plugins/obsidian-collapse-all-plugin|obsidian-collapse-all-plugin]]
-- [[02 - Community plugins/02.01 - Plugins/obsidian-command-alias-plugin|obsidian-command-alias-plugin]]
+- [[02 - Community plugins/02.01 - Plugins/obsidian-command-aliases-plugin|obsidian-command-aliases-plugin]]
```
Comment (kometenstaub) [inline 02 - Community plugins/02.01 - Plugins/02.01 - Plugins.md:135]: That was unintended.

File: 02 - Community plugins/02.01 - Plugins/dangerzone-writing-plugin.md [hunk 9]
```diff
@@ -9,10 +9,10 @@ publish: true
 
 %% ----- Badges ----- %%
 
-![GitHub all releases](https://img.shields.io/github/downloads/akaalias/dangerzone-writing-plugin/total?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub manifest version](https://img.shields.io/github/manifest-json/v/akaalias/dangerzone-writing-plugin?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub issues by-label](https://img.shields.io/github/issues/akaalias/dangerzone-writing-plugin/help%20wanted?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub Repo stars](https://img.shields.io/github/stars/akaalias/dangerzone-writing-plugin?color=573E7A&logo=github&style=for-the-badge)
+![GitHub all releases](https://img.shields.io/github/downloads/akaaliases/dangerzone-writing-plugin/total?color=573E7A&logo=github&style=for-the-badge)
```
Comment (kometenstaub) [inline 02 - Community plugins/02.01 - Plugins/dangerzone-writing-plugin.md:12]: These four were unintended.

File: 02 - Community plugins/02.01 - Plugins/dangerzone-writing-plugin.md [hunk 10]
```diff
@@ -23,7 +23,7 @@ publish: true
 
 # Dangerzone Writing
 
-Repository: [GitHub](https://github.com/akaalias/dangerzone-writing-plugin)
+Repository: [GitHub](https://github.com/akaaliases/dangerzone-writing-plugin)
```
Comment (kometenstaub) [inline 02 - Community plugins/02.01 - Plugins/dangerzone-writing-plugin.md:26]: This one as well.

File: 02 - Community plugins/02.01 - Plugins/extract-highlights-plugin.md [hunk 11]
```diff
@@ -9,10 +9,10 @@ publish: true
 
 %% ----- Badges ----- %%
 
-![GitHub all releases](https://img.shields.io/github/downloads/akaalias/extract-highlights-plugin/total?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub manifest version](https://img.shields.io/github/manifest-json/v/akaalias/extract-highlights-plugin?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub issues by-label](https://img.shields.io/github/issues/akaalias/extract-highlights-plugin/help%20wanted?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub Repo stars](https://img.shields.io/github/stars/akaalias/extract-highlights-plugin?color=573E7A&logo=github&style=for-the-badge)
+![GitHub all releases](https://img.shields.io/github/downloads/akaaliases/extract-highlights-plugin/total?color=573E7A&logo=github&style=for-the-badge)
```
Comment (kometenstaub) [inline 02 - Community plugins/02.01 - Plugins/extract-highlights-plugin.md:12]: Four.

File: 02 - Community plugins/02.01 - Plugins/extract-highlights-plugin.md [hunk 12]
```diff
@@ -23,7 +23,7 @@ publish: true
 
 # Extract Highlights
 
-Repository: [GitHub](https://github.com/akaalias/extract-highlights-plugin)
+Repository: [GitHub](https://github.com/akaaliases/extract-highlights-plugin)
```
Comment (kometenstaub) [inline 02 - Community plugins/02.01 - Plugins/extract-highlights-plugin.md:26]: One.

File: 02 - Community plugins/02.01 - Plugins/obsidian-command-alias-plugin.md [hunk 13]
```diff
@@ -1,18 +1,18 @@
...
 publish: true
 ---
 
 %% ----- Badges ----- %%
 
-![GitHub all releases](https://img.shields.io/github/downloads/yajamon/obsidian-command-alias-plugin/total?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub manifest version](https://img.shields.io/github/manifest-json/v/yajamon/obsidian-command-alias-plugin?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub issues by-label](https://img.shields.io/github/issues/yajamon/obsidian-command-alias-plugin/help%20wanted?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub Repo stars](https://img.shields.io/github/stars/yajamon/obsidian-command-alias-plugin?color=573E7A&logo=github&style=for-the-badge)
+![GitHub all releases](https://img.shields.io/github/downloads/yajamon/obsidian-command-aliases-plugin/total?color=573E7A&logo=github&style=for-the-badge)
```
Comment (kometenstaub) [inline 02 - Community plugins/02.01 - Plugins/obsidian-command-alias-plugin.md:12]: Four.

File: 02 - Community plugins/02.01 - Plugins/obsidian-command-alias-plugin.md [hunk 14]
```diff
@@ -21,11 +21,11 @@ publish: true
 
 %% ----- Do not edit this section ----- %%
 
-# Command Alias
+# Command aliases
```
Comment (kometenstaub) [inline 02 - Community plugins/02.01 - Plugins/obsidian-command-alias-plugin.md:24]: Next three from here.

File: 02 - Community plugins/02.01 - Plugins/obsidian-extract-pdf-highlights.md [hunk 15]
```diff
@@ -9,10 +9,10 @@ publish: true
 
 %% ----- Badges ----- %%
 
-![GitHub all releases](https://img.shields.io/github/downloads/akaalias/obsidian-extract-pdf-highlights/total?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub manifest version](https://img.shields.io/github/manifest-json/v/akaalias/obsidian-extract-pdf-highlights?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub issues by-label](https://img.shields.io/github/issues/akaalias/obsidian-extract-pdf-highlights/help%20wanted?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub Repo stars](https://img.shields.io/github/stars/akaalias/obsidian-extract-pdf-highlights?color=573E7A&logo=github&style=for-the-badge)
+![GitHub all releases](https://img.shields.io/github/downloads/akaaliases/obsidian-extract-pdf-highlights/total?color=573E7A&logo=github&style=for-the-badge)
```
Comment (kometenstaub) [inline 02 - Community plugins/02.01 - Plugins/obsidian-extract-pdf-highlights.md:12]: Four.

File: 02 - Community plugins/02.01 - Plugins/obsidian-extract-pdf-highlights.md [hunk 16]
```diff
@@ -23,7 +23,7 @@ publish: true
 
 # PDF Highlights
 
-Repository: [GitHub](https://github.com/akaalias/obsidian-extract-pdf-highlights)
+Repository: [GitHub](https://github.com/akaaliases/obsidian-extract-pdf-highlights)
```
Comment (kometenstaub) [inline 02 - Community plugins/02.01 - Plugins/obsidian-extract-pdf-highlights.md:26]: Also here.

File: 02 - Community plugins/02.01 - Plugins/obsidian-footnotes.md [hunk 17]
```diff
@@ -9,10 +9,10 @@ publish: true
 
 %% ----- Badges ----- %%
 
-![GitHub all releases](https://img.shields.io/github/downloads/akaalias/obsidian-footnotes/total?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub manifest version](https://img.shields.io/github/manifest-json/v/akaalias/obsidian-footnotes?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub issues by-label](https://img.shields.io/github/issues/akaalias/obsidian-footnotes/help%20wanted?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub Repo stars](https://img.shields.io/github/stars/akaalias/obsidian-footnotes?color=573E7A&logo=github&style=for-the-badge)
+![GitHub all releases](https://img.shields.io/github/downloads/akaaliases/obsidian-footnotes/total?color=573E7A&logo=github&style=for-the-badge)
```
Comment (kometenstaub) [inline 02 - Community plugins/02.01 - Plugins/obsidian-footnotes.md:12]: Four.

File: 02 - Community plugins/02.01 - Plugins/obsidian-footnotes.md [hunk 18]
```diff
@@ -23,7 +23,7 @@ publish: true
 
 # Footnote Shortcut
 
-Repository: [GitHub](https://github.com/akaalias/obsidian-footnotes)
+Repository: [GitHub](https://github.com/akaaliases/obsidian-footnotes)
```
Comment (kometenstaub) [inline 02 - Community plugins/02.01 - Plugins/obsidian-footnotes.md:26]: One.

File: 02 - Community plugins/02.01 - Plugins/obsidian-journey-plugin.md [hunk 19]
```diff
@@ -9,10 +9,10 @@ publish: true
 
 %% ----- Badges ----- %%
 
-![GitHub all releases](https://img.shields.io/github/downloads/akaalias/obsidian-journey-plugin/total?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub manifest version](https://img.shields.io/github/manifest-json/v/akaalias/obsidian-journey-plugin?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub issues by-label](https://img.shields.io/github/issues/akaalias/obsidian-journey-plugin/help%20wanted?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub Repo stars](https://img.shields.io/github/stars/akaalias/obsidian-journey-plugin?color=573E7A&logo=github&style=for-the-badge)
+![GitHub all releases](https://img.shields.io/github/downloads/akaaliases/obsidian-journey-plugin/total?color=573E7A&logo=github&style=for-the-badge)
```
Comment (kometenstaub) [inline 02 - Community plugins/02.01 - Plugins/obsidian-journey-plugin.md:12]: Four.

File: 02 - Community plugins/02.01 - Plugins/obsidian-journey-plugin.md [hunk 20]
```diff
@@ -23,7 +23,7 @@ publish: true
 
 # Journey
 
-Repository: [GitHub](https://github.com/akaalias/obsidian-journey-plugin)
+Repository: [GitHub](https://github.com/akaaliases/obsidian-journey-plugin)
```
Comment (kometenstaub) [inline 02 - Community plugins/02.01 - Plugins/obsidian-journey-plugin.md:26]: One.

File: 02 - Community plugins/02.01 - Plugins/pdf-to-markdown-plugin.md [hunk 21]
```diff
@@ -9,10 +9,10 @@ publish: true
 
 %% ----- Badges ----- %%
 
-![GitHub all releases](https://img.shields.io/github/downloads/akaalias/obsidian-extract-pdf/total?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub manifest version](https://img.shields.io/github/manifest-json/v/akaalias/obsidian-extract-pdf?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub issues by-label](https://img.shields.io/github/issues/akaalias/obsidian-extract-pdf/help%20wanted?color=573E7A&logo=github&style=for-the-badge)   
-![GitHub Repo stars](https://img.shields.io/github/stars/akaalias/obsidian-extract-pdf?color=573E7A&logo=github&style=for-the-badge)
+![GitHub all releases](https://img.shields.io/github/downloads/akaaliases/obsidian-extract-pdf/total?color=573E7A&logo=github&style=for-the-badge)
```
Comment (kometenstaub) [inline 02 - Community plugins/02.01 - Plugins/pdf-to-markdown-plugin.md:12]: Four.

File: 02 - Community plugins/02.01 - Plugins/pdf-to-markdown-plugin.md [hunk 22]
```diff
@@ -23,7 +23,7 @@ publish: true
 
 # PDF to Markdown
 
-Repository: [GitHub](https://github.com/akaalias/obsidian-extract-pdf)
+Repository: [GitHub](https://github.com/akaaliases/obsidian-extract-pdf)
 Mobile compatible: [[Mobile-compatible plugins|Yes]]
```
Comment (kometenstaub) [inline 02 - Community plugins/02.01 - Plugins/pdf-to-markdown-plugin.md:27]: One.

File: 02 - Community plugins/02.02 - Categories/02.02 - Categories.md [hunk 23]
```diff
@@ -60,7 +60,7 @@ tags:
 
 1. Open the category note(s) that you want to add the plugin to from the list above.
 2. Add a new bullet in alphabetical order for the new plugin
-3. Add a link to the plugin in question using its alias, e.g. `[[obsidian-dataview|Dataview]]`
+3. Add a link to the plugin in question using its aliases, e.g. `[[obsidian-dataview|Dataview]]`
```
Comment (kometenstaub) [inline 02 - Community plugins/02.02 - Categories/02.02 - Categories.md:63]: This one as well.

File: 02 - Community plugins/02.02 - Categories/Automation.md [hunk 24]
```diff
@@ -1,5 +1,5 @@
 ---
-aliases:
+aliaseses:
```
Comment (kometenstaub) [inline 02 - Community plugins/02.02 - Categories/Automation.md:2]: Why did it change `aliases` to `aliaseses`?

File: 02 - Community plugins/02.02 - Categories/Shortcut.md [hunk 25]
```diff
@@ -28,7 +28,7 @@ Hotkeys
 - [[obsidian-underline|Underline]]
 - [[obsidian-code-copy|Code Copy]]
 - [[editor-commands-remap|Editor Commands Remap]]
-- [[obsidian-command-alias-plugin|Command Alias]]
+- [[obsidian-command-aliases-plugin|Command aliases]]
```
Comment (kometenstaub) [inline 02 - Community plugins/02.02 - Categories/Shortcut.md:31]: One.

**Review by kometenstaub — CHANGES_REQUESTED** [review]
I think you didn't search and replace `alias:` with `aliases:` but rather `alias` with `aliases` which resulted in a lot of false positives. I suggest to hard-reset to the commit before.

**Review by kometenstaub — APPROVED** [review]
I didn't find any false positives. :+1:

**Pull request merged**